apps/web/src/utils/buildURL.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+/**
+ * API 요청을 보낼 때 URL을 생성하는 util 함수입니다.
```

Comment:
p3) @description으로 jsdoc을 달아줘도 좋을 것 같아요!
```suggestion
 * @description API 요청을 보낼 때 URL을 생성하는 util 함수입니다.
```